app/src/main/java/com/saotome/imed_app/ui/fragments/DrugListFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.saotome.imed_app.ui.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.saotome.imed_app.databinding.FragmentDrugListBinding
+import com.saotome.imed_app.model.Drug
+import com.saotome.imed_app.ui.DrugListAdapter
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DrugListFragment : Fragment() {
+
+    private var _binding: FragmentDrugListBinding? = null
+    private val drugList = arrayListOf<Drug>()
+    // This property is only valid between onCreateView and
```

Comment:
Does this comment still make sense? I think you removed the line below it.

Reply:
No it doesn't. Thanks for the reminder :)
ac033076add9121739da75bacc1053902964c6a8